web-ui/src/components/feelings/Feelings.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+
+const Feelings = (props) => {
+  const { onSelect, message } = props;
```

Comment:
Any particular reason you do this props selector on a new line rather than inline in the method definition?  It's super minor, but whenever I see props passed in rather than a selector, it makes me feel like I need to look through the whole component to make sure there aren't other things that are passed in that I'm not noticing.

It might also help to define a propTypes.

Reply:
Also a displayName...